core/src/main/scala/kafka/coordinator/GroupCoordinator.scala

```diff
@@ -256,6 +258,7 @@ class GroupCoordinator(val brokerId: Int,
 
           case AwaitingSync =>
             group.get(memberId).awaitingSyncCallback = responseCallback
+            // FIXME: is below needed?
```

Comment:
Are you referring to the `completeAndScheduleNextHeartbeatExpiration`? Presumably it isn't strictly required, but doesn't it just allow us to treat the sync as an implicit heartbeat, which seems like a good thing?